### ISSUE TITLE
For field agent,  sent for review >audit record page is incorrect for any incomplete birth/death 

### ISSUE THREAD

**Issue by Fariha-Amin** [opencrvs/opencrvs-core#3228]
**Description:**
For field agent, sent for review -audit record page is incorrect for any incomplete birth/death . Place of birth/death and informant's information's should be hidden.

**Steps:**

1. Login as  field agent 
2. declare a birth/death event
3. submit an incomplete birth/death  application
4. goto sent for review page
5. click the application
6. observe "DOB/DOD" and "informant"

**Actual Result:**
DOB/DOD and informants information is visible.

**Expected Result:**
DOB/DOD and informants information shouldn't be visible. 

**ScreenShot:**
![birth_incmp.PNG](https://images.zenhubusercontent.com/6281e46d6f51be7532fe12bd/aea3c242-4328-4814-8e8a-008d7db4be65)

**Tested on:**
https://login.farajaland-qa.opencrvs.org/

**Desktop:**
OS: Windows 10
Browser: Chrome

**Comment by euanmillar** [comment]
Potential duplicate of https://github.com/opencrvs/opencrvs-core/issues/3231 after question is answered and unblocked

**Comment by Fariha-Amin** [comment]
@anny320  passed, can be closed

screenshot:
![d](https://user-images.githubusercontent.com/105624748/171582872-05035596-b6b1-40b3-8cc6-45a6261cbcaf.PNG)

**Comment by anny320** [comment]
completed
